File: src/com/classe/solaire/Presence.java
package com.classe.solaire;

import java.lang.Object;

public class Presence {
  private Object idbatiment;

  private Object datepresence;

  private Object tranchehoraire;

  private Object nbreleve;

  public Presence(Object idbatiment, Object datepresence, Object tranchehoraire, Object nbreleve) {
    this.idbatiment = idbatiment;
    this.datepresence = datepresence;
    this.tranchehoraire = tranchehoraire;
    this.nbreleve = nbreleve;
  }

  public Object getIdbatiment() {
    return idbatiment;
  }

  public void setIdbatiment(Object idbatiment) {
    this.idbatiment = idbatiment;
  }

  public Object getDatepresence() {
    return datepresence;
  }

  public void setDatepresence(Object datepresence) {
    this.datepresence = datepresence;
  }

  public Object getTranchehoraire() {
    return tranchehoraire;
  }

  public void setTranchehoraire(Object tranchehoraire) {
    this.tranchehoraire = tranchehoraire;
  }

  public Object getNbreleve() {
    return nbreleve;
  }

  public void setNbreleve(Object nbreleve) {
    this.nbreleve = nbreleve;
  }
}

File: src/com/cs/model/solaire/Presence.cs
using System;
namespace src.com.cs.models {
public class Presence {
 {
    private object idbatiment;
    private object datepresence;
    private object tranchehoraire;
    private object nbreleve;

    public Presence(object idbatiment, object datepresence, object tranchehoraire, object nbreleve) {
        this.idbatiment = idbatiment;
        this.datepresence = datepresence;
        this.tranchehoraire = tranchehoraire;
        this.nbreleve = nbreleve;
    }
    public object Idbatiment {
        get { return idbatiment; }
        set { idbatiment = value; }
    }
    public object Datepresence {
        get { return datepresence; }
        set { datepresence = value; }
    }
    public object Tranchehoraire {
        get { return tranchehoraire; }
        set { tranchehoraire = value; }
    }
    public object Nbreleve {
        get { return nbreleve; }
        set { nbreleve = value; }
    }
}
    }
}